initializr-generator/src/main/java/io/spring/initializr/generator/ProjectGenerator.java

```diff
@@ -22,22 +22,12 @@
 import java.io.IOException;
 import java.io.OutputStream;
 import java.nio.charset.Charset;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.Comparator;
-import java.util.HashMap;
-import java.util.LinkedHashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
Please refrain from reformating the code or the imports. Can you rework your PR to only submit the necessary changes?